Revert/Sources/HomeViewController.swift

```diff
@@ -63,8 +59,7 @@ final class HomeViewController: UITableViewController {
   private var collection = CollectableCollection<HomeItem>(items: .Home)
   private var dataSource: DataSource<HomeItem, HomeCell>
   private var isSplitViewControllerCollapsed: Bool {
-    // If self.splitViewController is nil then we're on iOS7 on phone, i.e equivalent to collapsed mode
-    return self.splitViewController?.backwardCompatibleCollapsed ?? true
+    return self.splitViewController?.collapsed ?? true
   }
```

Comment:
> Are we happy to rely on this implementation detail […] ?

No, I'm not. I think the API is clear here — `splitViewController` _can_ be `.None`. I prefer to be explicit (and also to follow what's documented in the headers).

Trust but verify, right?